huaweicloud/services/vpn/resource_huaweicloud_vpn_gateway.go

```diff
@@ -111,11 +111,11 @@ func ResourceGateway() *schema.Resource {
 			"flavor": {
 				Type:        schema.TypeString,
 				Optional:    true,
-				Default:     "V300",
 				ForceNew:    true,
+				Computed:    true,
 				Description: `The flavor of the VPN gateway.`,
 				ValidateFunc: validation.StringInSlice([]string{
-					"V1G", "V300",
+					"V1G", "V300", "Professional1", "Professional2",
```

Comment:
according to the [API docs](https://support.huaweicloud.com/api-vpn/vpn_api_0014.html),  the `Basic` value should also be allowed?

Reply:
done